### PR TITLE
Add genericParameters to BPM start BC creation endpoint

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

**Comment by jbartece** [comment]
@jsenko  Review, please. I suppose it'll require also change in the BCCreation process.

**Comment by jbartece** [comment]
retest this please

**Comment by jbartece** [comment]
retest this please

File: rest-model/src/main/java/org/jboss/pnc/rest/restmodel/bpm/BpmBuildConfigurationCreationRest.java [hunk 1]
```diff
@@ -91,4 +94,8 @@
     @Getter
     @Setter
     private Set<Integer> buildConfigurationSetIds;
+    
+    @Getter
+    @Setter
+    private Map<String, String> genericParameters ;
```
Comment (michalszynkiewicz) [inline rest-model/src/main/java/org/jboss/pnc/rest/restmodel/bpm/BpmBuildConfigurationCreationRest.java]: could you remove the space between `s` and `;`? (btw in some next change you'll make).

**Pull request merged**